moira_client/models/trigger.py

```diff
@@ -262,6 +262,54 @@ def check_exists(self):
                 set(self.tags) == set(trigger.tags):
                 return trigger
 
+    def get_metrics(self, _from, to):
```

Comment:
It looks weird that we have `_from` and `to`. I would like to see something more consistent in naming. Let's probably use `start` and `end` or `time_from` and `time_to`.